### PR TITLE
Add a link to Special Pages definition in Client's technical context

### PR DESCRIPTION
**Pull request by tzhelyazkova**
Adding a link to the Glossary definition of a Special Page should clear all confusion.

### CODE REVIEW

**Review by Silvan-WMDE — COMMENTED** [review]
tbh, it isn't immediately clear to me, why and how Special Pages provide "user interfaces for viewing Entity meta data". People who, like me, expect an explanation on that in the Glossary entry will be disappointed, because the Glossary only describes the very broad concept of MediaWiki Special Pages in general. So, contrary to the feedback we are not actually digging a little deeper here, but "digging up" instead. :-)

But I'm curious to hear what others say about the feedback item itself and this solution here. Don't want to block it.

**Comment by tzhelyazkova** [comment]
> tbh, it isn't immediately clear to me, why and how Special Pages provide "user interfaces for viewing Entity meta data". People who, like me, expect an explanation on that in the Glossary entry will be disappointed, because the Glossary only describes the very broad concept of MediaWiki Special Pages in general. So, contrary to the feedback we are not actually digging a little deeper here, but "digging up" instead. :-)
> 
> But I'm curious to hear what others say about the feedback item itself and this solution here. Don't want to block it.

I went back and forth between sections trying to come up with something more than linking the Glossary, but the current way this is explained seemed enough to me. If the reader wants to understand the term Special Page, the glossary is the place to read up on it. If the reader wants to understand Special Pages in this diagram's context, they should look at the appropriately linked building block view below. I don't have any other ideas how to change this.

File: systems/WikibaseClient/03-Context_and_Scope.md [hunk 1]
```diff
@@ -21,6 +21,6 @@ WikibaseClient is a "plugin" (= extension) that lives inside [MediaWiki](../over
...
 | [UpdateRepoHookHandler](./05-Building_Block_View.md#linked-site-page-changes) | Notifies the [Repo](../overview/12-Glossary.md#wikibase-repository) to update the connected Item when a page is moved or deleted                                                                              |
 | [Data Bridge](./05-Building_Block_View.md#client-side-item-edits)             | A frontend component enabling [Repo](../overview/12-Glossary.md#wikibase-repository) edits on the Client via the Repo API                                                                                     |
-| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data                                                                                                                                                                  |
+| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data. For a full description see [Special Pages](../overview/12-Glossary.md#special-page)                                                                             |
```
Comment (addshore) [inline systems/WikibaseClient/03-Context_and_Scope.md:24]: For extra clarity about what to expect I think this should be reworded to include text saying it's linking you to the glossary for a general definition of the term special page. When reading it now it seems slightly confusing as there are 2 textual links with the text "Special Pages" that link to different places without saying which goes where.

Not sure of the best text, but even (See what a Special Pages is in the glossary) or something might make sense.

Comment (tzhelyazkova) [reply systems/WikibaseClient/03-Context_and_Scope.md]: added

**Review by addshore — APPROVED** [review]

**Pull request merged**